### DISCUSSION TITLE
how to discover in what mode memory is working? (dual, single, quad channel)

### DISCUSSION THREAD

**Discussion by Frambooisier** [memtest86plus/memtest86plus#231] [Q&A] [answered]
I see that in older version of memtest there was a vaule displayed on the screen that would tell you whats the current width like "128-bit". Can it be done with new version of memtest?

**Comment by debrouxl** [comment, marked as answer]
Indeed, the current version of memtest86+ doesn't support it anymore: Martin Whitaker's primary focus for pcmemtest, which has become the new memtest86+, was to build a solid foundation (clean, minimal code base) for reliable memory tests.
Adding live CPU frequency / IMC / ECC / EDAC reading code on top of that foundation is possible; the tracking discussion for this enhancement is #165 :)
